doc/examples/cloud-config-seed-random.txt

```diff
@@ -1,32 +0,0 @@
-#cloud-config
-#
-# random_seed is a dictionary.
-#
-# The config module will write seed data from the datasource
-# to 'file' described below.
-#
-# Entries in this dictionary are:
-#   file:     the file to write random data to (default is /dev/urandom)
-#   data:     this data will be written to 'file' before data from
-#             the datasource
-#   encoding: this will be used to decode 'data' provided.
-#             allowed values are 'encoding', 'raw', 'base64', 'b64'
-#             'gzip', or 'gz'.  Default is 'raw'
-#
-#   command:  execute this command to seed random.
-#             the command will have RANDOM_SEED_FILE in its environment
-#             set to the value of 'file' above.
-#   command_required: default False
-#             if true, and 'command' is not available to be run
-#             then exception is raised and cloud-init will record failure.
-#             Otherwise, only debug error is mentioned.
-#
-# Note: command could be ['pollinate',
-#                         '--server=http://local.pollinate.server']
-# which would have pollinate populate /dev/urandom from provided server
```

Comment:
I don't like losing documented use-cases from examples and not representing them in module.Meta.examples. Given that the examples page is the page with the most daily hits, I don't want to diminish the value of that page without providing supplemental/comparable examples somewhere else. Please include a supplemental example that explains this use-case in cc_seed_random.

Reply:
Heh I just re-looked at full rendered docs and see that this example is already in cc_seed_random's MODULE_DESCRIPTION. Maybe we can just move that example and introductory header comment down into Meta['examples']